numpy/_core/include/numpy/ufuncobject.h

```diff
@@ -191,6 +224,12 @@ typedef struct _tagPyUFuncObject {
         /* A PyListObject of `(tuple of DTypes, ArrayMethod/Promoter)` */
         PyObject *_loops;
     #endif
+    #if NPY_FEATURE_VERSION >= NPY_2_1_API_VERSION
```

Comment:
Just noticed, but just noting to not forget:  `NPY_2_1_API_VERSION` isn't defined yet I think.  But don't worry about it, I can just do it (maybe even later today) since evertying else should be good.